content/v2.0/introduction/getting-started.md

```diff
@@ -138,29 +162,1243 @@ Visit your application in the browser at [http://localhost:2300](http://localhos
+
+Hanami.app.prepare(:persistence)
+DatabaseCleaner[:sequel, db: Hanami.app["persistence.db"]]
+
+RSpec.configure do |config|
+  config.before(:suite) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.around(:each, type: :database) do |example|
+    DatabaseCleaner.cleaning do
+      example.run
+    end
+  end
+end
+```
+
+And then append the following line to `spec/spec_helper.rb`:
+
+```ruby
+require_relative "support/database_cleaner"
+```
+
+Finally, enable rom-rb's rake tasks for database migrations by appending the following to the `Rakefile`:
+
+```ruby
+# Rakefile
+
+require "rom/sql/rake_task"
+
+namespace :db do
+  task setup: :environment do
+    require "rom/core"
+    rom_config = ROM::Configuration.new(:sql, Hanami.app["settings"].database_url)
+    ROM::SQL::RakeSupport.env = rom_config
+  end
+end
+```
+
+<p class="notice">
+  Hanami's 2.1 release, slated for early 2023, will bring persistence as a first class feature, after which none of the above set up will be required.
+</p>
+
+### Creating a books table
+
+With persistence ready, we can now create a books table.
+
+To create a migration run `bundle exec rake db:create_migration[create_books]`:
+
+```shell
+bundle exec rake db:create_migration[create_books]
+<= migration file created db/migrate/20221113050928_create_books.rb
+```
+
+Edit the migration file in order to create a books table with title and author columns and a primary key:
+
+```ruby
+# db/migrate/20221113050928_create_books.rb
 
-## Commands
+# frozen_string_literal: true
 
-Hanami ships with useful commands for things like starting a console, inspecting routes and generating code.
+ROM::SQL.migration do
+  change do
+    create_table :books do
+      primary_key :id
+      column :title, :text, null: false
+      column :author, :text, null: false
+    end
+  end
+end
+```
 
-To list available commands, run:
+Migrate both the development and test databases:
 
 ```shell
-bundle exec hanami --help
+bundle exec rake db:migrate
+HANAMI_ENV=test bundle exec rake db:migrate
+```
+
+Lastly, let's add a rom-rb relation to allow our application to interact with our books table. Create the following file at `lib/bookshelf/persistence/relations/books.rb`:
+
+```ruby
+# lib/bookshelf/persistence/relations/books.rb
+
+# frozen_string_literal: true
+
+module Bookshelf
+  module Persistence
+    module Relations
+      class Books < ROM::Relation[:sql]
+        schema(:books, infer: true)
+      end
+    end
+  end
+end
+```
+
+## Listing books
+
+With our books table ready to go, let's adapt our books index spec to expect an index of persisted books:
+
+```ruby
+# frozen_string_literal: true
+
+RSpec.describe "GET /books", type: [:request, :database] do
+  let(:books) { app["persistence.rom"].relations[:books] }
+
+  before do
+    books.insert(title: "Refactoring", author: "Martin Fowler")
+    books.insert(title: "Test Driven Development", author: "Kent Beck")
```

Comment:
It'd be really nice to have a non-male author as an example too. Maybe "POODR" by Sandi Metz?

Reply:
Done, thanks @solnic.